encrypted-media-respec.html

```diff
@@ -1724,11 +1714,12 @@
           <p class="note">The returned promise is resolved when the request has been processed, and the <a def-id="closed"></a> attribute promise is resolved when the session is closed.</p>
 
           <ol class="method-algorithm">
-            <li><p>If the <a def-id="session-close-algorithm"></a> algorithm has been run on this object, return a resolved promise.</p></li>
+            <li><p>If the <a def-id="session-closed-algorithm"></a> algorithm has been run on this object, return a resolved promise.</p></li>
             <li><p>If this object's <var>callable</var> value is false, return a promise rejected with an <a def-id="InvalidStateError"></a>.</p></li>
             <li><p>Let <var>promise</var> be a new promise.</p></li>
             <li><p>Run the following steps in parallel:</p>
               <ol>
+                <!--- Consider whether to replace the following with a call to Close Session -->
```

Comment:
See my comment at line 2083.

If this does remain unresolved in this PR (I don't think it should), add "TODO:" so we don't forget about it.